src/core/BaseUtilities.ts

```diff
@@ -23,17 +22,12 @@ export class BaseUtilities {
     public get database(): Database { return this.client.database; }
     public get logger(): Logger { return this.client.logger; }
     public get config(): Configuration { return this.client.config; }
-    public readonly translator: FileSystemTranslationSource;
+    public readonly translator: CrowdinTranslationSource;
 
     public constructor(
         public readonly client: BaseClient
     ) {
-        this.translator = new FileSystemTranslationSource(
-            path.join(
-                path.dirname(require.resolve('@blargbot/res/package')),
-                'i18n'
-            )
-        );
+        this.translator = new CrowdinTranslationSource('a713aad8fe135bf923f9587yoka');
```

Comment:
what's that key there?

Reply:
Thats the public distribution key, used to say what bundle of files from the CDN we want

Reply:
I could put it in the config, but theres no need to keep it secret and I doubt this will ever change